proposals/getposition-sidepanel.md

```diff
@@ -0,0 +1,130 @@
+# Add API to Query Side Panel Position
+
+**Summary**
+
+Proposal to introduce a programmatic way to query the Side panel position.
+
+**Document Metadata**
+
+**Author:** [hharshas](https://github.com/hharshas)
+
+**Sponsoring Browser:** Chromium
+
+**Contributors:** [hharshas](https://github.com/hharshas)
+([Summer Of Code](https://summerofcode.withgoogle.com/)), mentor: oliverdunk, solomonkinard
+
+**Created:** 2025-05-27
+
+**Related Issues:** [#chromium](https://issues.chromium.org/issues/406511291)
+
+## Motivation
+
+### Objective
+
+* At `chrome://settings`, it is possible to change the "Side panel position"
```

Comment:
We should be careful about when we reference Chrome specifically, since we want to motivate why this makes sense across all browsers. I would just mention this as one approach, for example:

> Browsers may offer users the ability to change which side of the browser the side panel opens on. In Chrome, this is exposed at chrome://settings.